picasso/src/main/java/com/squareup/picasso3/Picasso.java

```diff
@@ -136,8 +136,10 @@ void onImageLoadFailed(@NonNull Picasso picasso, @NonNull Uri uri,
   private final @Nullable okhttp3.Cache closeableCache;
   final PlatformLruCache cache;
   final Stats stats;
+  final ImageDecoderFactory imageDecoderFactory;
```

Comment:
where is this used? it should get passed to the RequestCreate/Builder thing.

Reply:
This is used by the `RequestCreator` and `Request.Builder`.  The convention seems to be passing the whole `Picasso` instance into the `RequestCreator` and having it access the properties on it, like `defaultBitmapConfig`, `dispatcher`, `cache`, etc.